.github/workflows/build.yml

```diff
@@ -16,7 +16,7 @@ permissions:
   contents: write
 
 env:
-  version: m126-6fd3120c1b-1
+  version: m133-60b586b703-1
```

Comment:
The plan was to update to m132 (LTS).

Also where I can review the merge conflicts resolution result?

Reply:
> Also where I can review the merge conflicts resolution result?
@MatkovIvan where you've reviewed them before? The procedure as I've understand it (based on our document) is about resolving this conflicts and then updating the skiko branch. 

I don't like this procedure, I do think there's a room for improvement in current workflow and definitely will be happy if we'll have more established, thorough and transparent procedure - so I'm not trying to brush off your question, I'm asking whether there's an established procedure as of now?